models/Snippet.js

```diff
@@ -0,0 +1,25 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var SnippetSchema = new Schema({
```

Comment:
I might be missing something, but where are we storing the actual contents of the snippet? Also, secondary thought that I had when I was doing the mockups last night... would it be helpful to give the user the option to add a description to the snippet for context? I added it that way to the mockup, just to give an idea of how it might look. Could be a terrible idea, but it seems like it would make sense and it would keep the title concise and to the point.

Reply:
I've added content and description fields.